MAINTAINERS

```diff
@@ -189,6 +189,7 @@ libquadmath		Tobias Burnus		<tobias@codesourcery.com>
 libquadmath		Jakub Jelinek		<jakub@redhat.com>
 libvtv			Caroline Tice		<cmtice@google.com>
 libphobos		Iain Buclaw		<ibuclaw@gdcproject.org>
+libgrust		All Rust maintainers
```

Comment:
Please say "Rust [front end] maintainers", and sort in alphabetically in `lib[...]` (even if not generally done consistently...).